### DISCUSSION TITLE
Latest release has invalid syntax

### DISCUSSION THREAD

**Discussion by gennadigennadigennadi** [deployphp/deployer#3925] [Bugs] [answered]
### Deployer Version

v7.5.0

### Target OS

Docker Image

### Which PHP version are you using?

PHP 7.4

### Steps to reproduce

Server.php has a trailing comma inside the constructor and that's not a valid php7.4 syntax.

<img width="572" alt="Screenshot 2024-10-22 at 11 00 57" src="https://github.com/user-attachments/assets/2e18aabb-74c0-41dd-b661-0c495521bc49">

**Comment by antonmedv** [comment, marked as answer]
Fixed in 7.5.11.
